backend/app/controllers/game_configurations_controller.rb

```diff
@@ -17,6 +17,10 @@ def show
   def create
     @game_configuration = GameConfiguration.new(game_configuration_params)
 
+    if !(game_configuration_params[:input_set] =~ /\A([A-Za-z]+ ){2,}[A-Za-z]+\z/) then 
```

Comment:
Could we get a couple of tests for this? It would be nice just to have as a sanity check.

Reply:
I wrote a couple tests, including 1 to check for uniqueness in adding new records. Everything in that file passes, but considering that stuff was already broken before I started adding tests (because of the database validations) I bet that we are going to have to make changes to other test files to also make them run perfectly.

Reply:
Is that something we should peel out into a separate story then?